kermit-test/build.gradle.kts

```diff
@@ -12,43 +12,14 @@
  */
 
 plugins {
-    id("com.android.library")
-    kotlin("multiplatform")
+    buildsrc.convention.`android-library`
+    buildsrc.convention.`kotlin-multiplatform`
+    buildsrc.convention.`publish-kotlin-multiplatform`
 }
```

Comment:
This PR is a quick demo, so I haven't updated all of the subprojects. Here's one I picked at random to show how the buildSrc plugins can be applied, and how the config can be removed :)